extensions/binary_glTF/README.md

```diff
@@ -1,10 +1,12 @@
-# CESIUM_binary_glTF
+# binary_glTF
```

Comment:
I changed it back for now.
Is there a schedule for moving to binary_glTF? Is this the deadline end of september?